### PR TITLE
Change all parameters to follow snake_case format

### PR DESCRIPTION
**Pull request by drpatelh**
Many thanks to contributing to nf-core/chipseq!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/chipseq branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/chipseq)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [x] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [x] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/chipseq/tree/master/.github/CONTRIBUTING.md

* Fixed bug in UpSetR plots not being rendered properly
* Changed `--design` to `--input`

### CODE REVIEW

**Comment by ewels** [comment]
Nice! What do you think about having the citations markdown file in the root of the repo and in capitals - `CITATIONS.md`. Then it’s easy to find, alongside the license and readme..

Also, would it be easy to also have a bibtex file? (Maybe we should have a txt file of DOIs / PMIDs and a script to generate the two files?)

**Comment by drpatelh** [comment]
Renamed citations file as you suggested @ewels 👍 

Im still trawling the web for a robust solution to simplify providing a file that we can use to generate bibtex, etc. Is it ok if I merge this for now?

File: README.md [hunk 1]
```diff
@@ -96,3 +96,6 @@ If you use nf-core/chipseq for your analysis, please cite it using the following
 
 You can cite the `nf-core` pre-print as follows:  
 > Ewels PA, Peltzer A, Fillinger S, Alneberg JA, Patel H, Wilm A, Garcia MU, Di Tommaso P, Nahnsen S. **nf-core: Community curated bioinformatics pipelines**. *bioRxiv*. 2019. p. 610741. [doi: 10.1101/610741](https://www.biorxiv.org/content/10.1101/610741v1).
+
+An extensive list of references for the tools used by the pipeline can be found in the [citation](CITATIONS.md) file.
```
Comment (ewels) [inline README.md]:
```suggestion
An extensive list of references for the tools used by the pipeline can be found in the [`CITATIONS.md`](CITATIONS.md) file.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * Capitalised process names
 * Add quick start information to main README
 * Update template to tools `1.7`
-* Bump Nextflow version to `19.04.0`
+* Add `--trim_nextseq` parameter
```
Comment (ewels) [inline CHANGELOG.md]:
```suggestion
* Add `--trim_nextseq` parameter
* Added `CITATIONS.md` file
```

**Pull request merged**